hackertracker/HTScheduleTableViewController.swift

```diff
@@ -12,13 +12,64 @@ import CoreData
 class BaseScheduleTableViewController: UITableViewController {
     
     var eventSections : [[Event]] = []
+    var syncAlert = UIAlertController(title: nil, message: "Syncing...", preferredStyle: .alert)
+    var data = NSMutableData()
 
     // TODO: Update for DC 25
     var days = ["2016-08-04", "2016-08-05", "2016-08-06", "2016-08-07"];
+    
+    func sync(sender: AnyObject) {
+        
+        let envPlist = Bundle.main.path(forResource: "Connections", ofType: "plist")
+        let envs = NSDictionary(contentsOfFile: envPlist!)!
+        
+        let tURL = envs.value(forKey: "URL") as! String
+        //NSLog("Connecting to \(tURL)")
+        let URL = Foundation.URL(string: tURL)
+        
+        let request = NSMutableURLRequest(url: URL!)
+        request.httpMethod = "GET"
+        
+        var queue = OperationQueue()
+        var con = NSURLConnection(request: request as URLRequest, delegate: self, startImmediately: true)
```

Comment:
NSURL Connection is fairly old. A newer class exists that makes this task much simpler. `NSURLSession` 

Here is an example

```
URLSession.shared.dataTask(with: request) { (data, response, error) in
  //ConnectionDidFinishLoading Logic here.
}
```